features/cellular/framework/AT/AT_CellularNetwork.cpp

```diff
@@ -97,6 +98,20 @@ void AT_CellularNetwork::read_reg_params_and_compare(RegistrationType type)
     int lac = -1, cell_id = -1, act = -1;
 
     read_reg_params(type, reg_status, lac, cell_id, act);
+    
+    switch (reg_status) {
```

Comment:
you could add this inside #if MBED_CONF_MBED_TRACE_ENABLE so when trace not enabled no extra code is generated